test/e2e/e2e.go

```diff
@@ -71,3 +77,21 @@ func WaitForScaleToZero(t *testing.T, deploymentName string, clients *test.Clien
 		cfg.ScaleToZeroGracePeriod*6,
 	)
 }
+
+func WaitForActivatorEndpoints(resources *v1a1test.ResourceObjects, clients *test.Clients) error {
+	// Wait for the endpoints to equalize.
+	return wait.Poll(250*time.Millisecond, time.Minute, func() (bool, error) {
```

Comment:
should `time.Minute` be a param to the function?

Reply:
I think this shouldn't change from test to test, so probably ok to have the same setting for all.

Reply:
You presume that all the transitions are based on the same workflow, but the workflow might be different.
For now though, it might not matter.